rhombus/scribblings/ref-map.scrbl

```diff
@@ -361,6 +363,23 @@ in an unspecified order.
 }
 
 
+@doc(
+  fun Map.append(map :: Map, ...) :: Map
+){
+
+ Functionally appends @rhombus(map)s, like the @rhombus(++) operator
+ (but without the special optimization). In case mappings of a key
```

Comment:
FWIW, I had trouble parsing `In case mappings of a key`. Maybe `When a key has a value in multiple given @rhombus(map)s,`?